tests/v2/validation/rbac/rbac_test.go

```diff
@@ -330,9 +358,21 @@ func (rb *RBTestSuite) TestRBAC() {
 				if strings.Contains(tt.role, "project") {
 					err := users.AddProjectMember(rb.client, rb.adminProject, rb.standardUser, tt.role)
 					require.NoError(rb.T(), err)
+
+					log.Info("Validating PRTB is created for the user.")
```

Comment:
I have to read through the code to know what the abbreviations stand for GRB, CRTB and PRTB. Since they are used interchangeably in the logs as-is, can we stick to only using the full name for clarity?

Reply:
Replaced the abbreviations "GRB, CRTB and PRTB" with their full names.